handler/README.md

```diff
@@ -120,7 +120,20 @@ Knot.x provides two node implementations:
 - **Action node** that represents simple steps in a graph such as integration with a data source
 - **Subtasks node** that is a list of unnamed tasks (subtasks) that are evaluated in parallel
 
+####### Node Processing Log
+
+Node processing produce flat logs in the following structure. 
+
+| Task | Node alias | Status | Transition | Node Log |
+|---|---|---|---|---|
+| TASK | A1 | SUCCESS | T1 | L-A1 |
```

Comment:
Can you please some example here. You can reuse the image from engine that presents processing states.
```
A
|     \
B    C
```

Reply:
You can also use `_success` transitions to be more accurate. You can describe also what is in the log when operation ends correctly or raises some exception.